platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/Utils.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
I kind of dislike the implied pattern of collecting all sorts of utility functions in this file. That won't scale well.
Granted, we could just refactor this in the future.

Reply:
then please give a suggestion, I will probably put all methods called by `AjaxDomEngine.java` into `AjaxEngine.ts`, so this is probably a non-issue